actions/getMumblePost.ts

```diff
@@ -0,0 +1,17 @@
+import { auth } from '@/app/api/auth/[...nextauth]/auth';
```

Comment:
Should this function be a server action? If yes, it needs to have the "use server" directive at the top of the file

Reply:
It works without. It has no react dependencies. Maybe we do not need to declare anything. I will check on that one,.

Reply:
It is probably just a function that is used on the server side. I suggest that we only put "server actions" functions into the `/actions` directory.